### PR TITLE
[Merged by Bors] - feat(OrderOfElement): add `isOfFinOrder_pow`

### PR DESCRIPTION
**Pull request by urkud**
Also add `IsOfFinOrder.of_pow` and a missing `@[to_additive]`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/GroupTheory/OrderOfElement.lean [hunk 1]
```diff
@@ -80,11 +80,29 @@ theorem not_isOfFinOrder_of_injective_pow {x : G} (h : Injective fun n : ℕ =>
   rw [h hnx] at hn_pos
   exact irrefl 0 hn_pos
 
+/-- 1 is of finite order in any monoid. -/
+@[to_additive (attr := simp) "0 is of finite order in any additive monoid."]
+theorem isOfFinOrder_one : IsOfFinOrder (1 : G) :=
```
Comment (YaelDillies) [inline Mathlib/GroupTheory/OrderOfElement.lean:85]: Could you take the opportunity to rename
```suggestion
theorem IsOfFinOrder.one : IsOfFinOrder (1 : G) :=
```
for anonymous dot notation?

Comment (urkud) [reply Mathlib/GroupTheory/OrderOfElement.lean]: Done (+added deprecated aliases).

**Comment by YaelDillies** [comment]
You seem to have missing an occurrence of `isOfFinOrder_one` in `GroupTheory.Torsion`. LGTM otherwise

maintainer merge

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**